### PR TITLE
OOPDEF-174: Form submission.

### PR DESCRIPTION
**Pull request by harrymaynard-maximus**
- [x] After these changes, the app was run and still works as expected
- [x] Tests for these changes were added (if applicable)
- [x] All existing unit tests were run and still pass

### CODE REVIEW

File: oop/src/views/ReviewPage.vue [hunk 1]
```diff
@@ -44,31 +50,58 @@ export default {
...
+            case '1': // Submission failed.
+              logService.logSubmission({ message: 'Submission failure', error: errorMessage }, applicationUuid, 'N/A');
+              this.navigateToSubmissionErrorPage();
+              break;
+            case '3': // System unavailable.
+              this.isSystemUnavailable = true;
+              logService.logSubmission({ message: 'Submission failure', error: errorMessage }, applicationUuid, 'N/A');
+              scrollToError();
+              break;
+          }
```
Comment (cyrilbravo) [inline oop/src/views/ReviewPage.vue:92]: Hi Harry, do we need to catch if the returnCode is 2 from the middleware?

**Review by cyrilbravo — CHANGES_REQUESTED** [review]
Just one comment in your PR but other than that, the code looks good.

File: oop/src/views/ReviewPage.vue (hunk 1 above)
Comment (harrymaynard-maximus) [reply oop/src/views/ReviewPage.vue:93]: It wasn't listed in the confluence doc, but I've added it in, and handling it as an error (since it's not '0').

**Comment by cyrilbravo** [comment]
Approved. Nice work, Harry.

**Review by cyrilbravo — APPROVED** [review]

**Pull request merged**